src/eos_gates.js

```diff
@@ -63,9 +113,47 @@ const Application = new Lang.Class({
         return this._konami.keyRelease(event);
     },
 
+    getHelpMessage: function() {
+        if (!this.replacement)
+            return _("You can install applications from our <a href='endlessm-app://eos-app-store'>App Store</a>.");
+
+        if (this.replacement.overrideHelpMessage)
+            return this.replacement.overrideHelpMessage;
+
+        let isEquivalentApp = !!this.replacement.replacementInfo;
+        let appName = !!this.replacement.replacementInfo ?
+                      this.replacement.replacementInfo.appName : this.replacement.appName;
+
+        if (this._alreadyHaveReplacement)
+            return _("However, you already have %s installed on this Computer").format("<b>%s</b>".format(appName));
```

Comment:
The capitalization of `Computer` looks odd; I guess we can tweak these messages later, but for now I'd just remove that last part from the message. Also, please factor out the bold app name, since it's also used later, and use single quotes for that, since the string is not translatable.